integration_test/factsengine_integration_test.go

```diff
@@ -0,0 +1,175 @@
+//go:build integration_test
+
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/trento-project/agent/internal/factsengine"
+	"github.com/trento-project/agent/internal/factsengine/adapters"
+	"github.com/trento-project/agent/internal/factsengine/entities"
+	"github.com/trento-project/contracts/pkg/events"
+)
+
+type IntegrationTestSuite struct {
```

Comment:
What do you think about a more specific naming like `FactsEngineIntegrationTestSuite`?